target/linux/ath79/dts/qca9557_iodata_wn-ac1600dgr2.dts

```diff
@@ -8,7 +8,7 @@
 
 / {
 	compatible = "iodata,wn-ac1600dgr2", "qca,qca9557";
```

Comment:
What about `compatible`, shouldn't you add `iodata,wn-ac1600dgr3` as well?

Reply:
OK, I'll add.